### PR TITLE
ROC-3359 Moment date conversion issue

### PR DESCRIPTION
**Pull request by dharmendrak**

### CODE REVIEW

File: src/main/app/claims/models/countyCourtJudgment.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export class CountyCourtJudgment {
                public paymentOption?: string,
                public paidAmount?: number,
                public repaymentPlan?: RepaymentPlan,
-               public payBySetDate?: Moment,
```
Comment (timja-kainos) [inline src/main/app/claims/models/countyCourtJudgment.ts]: this isn't right, we should be using a moment so we can do date work on it...

File: src/test/app/claims/models/settlement.ts [hunk 2]
```diff
@@ -51,7 +51,7 @@ describe('Settlement', () => {
       const offer: Offer = myInput.partyStatements[0].offer
       const actual: Settlement = new Settlement().deserialize(myInput)
 
-      expect(actual.getDefendantOffer().completionDate.format('YYYY-DD-MM')).to.be.eq(offer.completionDate)
+      expect(actual.getDefendantOffer().completionDate).to.be.eq(offer.completionDate)
```
Comment (dharmendrak) [inline src/test/app/claims/models/settlement.ts]: it is doing String comparison here, is it good to do for date?

File: src/main/app/claims/models/countyCourtJudgment.ts (hunk 1 above)
Comment (rlewan) [reply src/main/app/claims/models/countyCourtJudgment.ts]: 👍

File: src/main/app/claims/models/offer.ts [hunk 3]
```diff
@@ -1,20 +1,16 @@
-
-import { Moment } from 'moment'
-import { MomentFactory } from 'common/momentFactory'
-
 export class Offer {
   content: string
-  completionDate: Moment
+  completionDate: string
```
Comment (rlewan) [inline src/main/app/claims/models/offer.ts]: Same here.

File: src/main/app/claims/offerClient.ts [hunk 4]
```diff
@@ -4,42 +4,45 @@ import { Claim } from 'claims/models/claim'
 import { User } from 'idam/user'
 import { Offer as OfferForm } from 'features/offer/form/models/offer'
 import * as config from 'config'
-import { request } from 'client/request'
+import { request as requestPromiseApi, RequestPromiseAPI } from 'client/request'
 
 export const claimStoreApiUrl: string = `${config.get<string>('claim-store.url')}/claims`
 
 export class OfferClient {
+  constructor (private request: RequestPromiseAPI = requestPromiseApi) {
```
Comment (rlewan) [inline src/main/app/claims/offerClient.ts]: Why do you need to do this? I can't see any places where you initialise the client with something else.

**Comment by dharmendrak** [comment]
wrong fix

**Pull request closed without merging**